src/main/scala/wdl4s/AstTools.scala

```diff
@@ -286,6 +286,26 @@ object AstTools {
 
   def terminalMap(ast: Ast, source: WdlSource) = (findTerminals(ast) map {(_, source)}).toMap
 
+  def wdlSectionToStringMap(ast: Ast, node: String, wdlSyntaxErrorFormatter: WdlSyntaxErrorFormatter): Map[String, String] = {
+    ast.findAsts(node) match {
+      case a if a.isEmpty => Map.empty[String, String]
+      case a if a.size == 1 =>
+        // Yes, even 'meta {}' and 'parameter_meta {}' sections have RuntimeAttribute ASTs.
+        // In hindsight, this was a poor name for the AST.
```

Comment:
I believe @cjllanwarne is correct. As the underlying AST structure is tangential to this change and because I'd like to get this change in the release (and I don't have faith I'd manage to get both squared away) I filed [wdl issue 73](https://github.com/broadinstitute/wdl/issues/73).

I can't imagine there's a reason to *not* change the grammar for that (in fact it would more closely adhere to my reading of the spec) but we shouldn't change functionality like that w/o PO discussion, etc.